lib/platform/github/index.ts

```diff
@@ -1418,7 +1451,7 @@ async function tryPrAutomerge(
     return;
   }
 
-  if (!config.autoMergeAllowed) {
+  if (config.autoMergeAllowed === false) {
```

Comment:
This change means we're still trying automerge although don't know whether it's allowed or not. It potentially leads to lots of debug messages for GHE until explicitly disabling `platformAutomerge` via config.

Reply:
How much messages?

Reply:
It will print error for every new PR

Reply:
If a specific error occurs, it might be useful to set `config.autoMergeAllowed` to `false`.